DirectProgramming/DPC++FPGA/Tutorials/Features/ac_fixed/README.md

```diff
@@ -0,0 +1,291 @@
+   ```
+   You can also compile for a custom FPGA platform. Ensure that the board support package is installed on your system. Then run `cmake` using the command:
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=<board-support-package>:<board-variant>
+   ```
+
+2. Compile the design through the generated `Makefile`. The following build targets are provided, matching the recommended development flow:
+
+   * Compile for emulation (fast compile time, targets emulated FPGA device): 
+     ```
+     nmake fpga_emu
+     ```
+   * Generate the optimization report: 
+     ```
+     nmake report
+     ``` 
+   * Compile for FPGA hardware (longer compile time, targets FPGA device):
+     ```
+     nmake fpga
+     ``` 
+
+*Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX) do not yet support Windows*. Compiling to FPGA hardware on Windows* requires a third-party or custom Board Support Package (BSP) with Windows* support.<br>
+*Note:* If you encounter any issues with long paths when compiling under Windows*, you might have to create your `build` directory in a shorter path, for example `c:\samples\build`.  You can then run `cmake` from that directory, and provide `cmake` with the full path to your sample directory.
+ 
+### In Third-Party Integrated Development Environments (IDEs)
+
+You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the Visual Studio* IDE (in Windows*).
+For instructions, refer to [*Intel® oneAPI DPC++ FPGA Workflows on Third-Party IDEs*](https://software.intel.com/en-us/articles/intel-oneapi-dpcpp-fpga-workflow-on-ide)
+
+## Examining the Reports
+
+Locate the pair of `report.html` files in either:
+
+* **Report-only compile**:  `ac_fixed_report.prj`
+* **FPGA hardware compile**: `ac_fixed.prj`
+
+Scroll down on the Summary page of the report and expand the section titled **Compile Estimated Kernel Resource Utilization Summary**. Observe how the kernel `ConstructFromACFixed` consumes fewer resources than the kernel named `ConstructFromFloat`. Similarly, observe how the kernel named `CalculateWithACFixed` consumes fewer FPGA resources than `CalculateWithFloat`.
+
+## Running the Sample
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+
+   ```bash
+   ./ac_fixed.fpga_emu    # Linux
+   ac_fixed.fpga_emu.exe  # Windows
+   ```
+
+2. Run the sample on the FPGA device
+
+   ```bash
+   ./ac_fixed.fpga             # Linux
+   ```
+
+### Example of Output
+
+```txt
+Constructed from float:         3.6416015625
+Constructed from ac_fixed:      3.6416015625
+
+MAX DIFF for ac_fixed<10, 3, true>:  0.0078125
+MAX DIFF for float:                  9.53674e-07
+
+result(fixed point): 1
+result(float):       1
+
+result(fixed point): 0.992188
+result(float):       1
+
+result(fixed point): 1
+result(float):       1
+
+result(fixed point): 1
+result(float):       1
+
+result(fixed point): 0.992188
+result(float):       1
```

Comment:
For all these results, I don't know from this what they refer to, please add more print messages explaining what is beeing computed